firmware/chipcon_usb.c

```diff
@@ -36,6 +35,9 @@ __xdata u8   ep0req;
 __xdata u16  ep0len;
 __xdata u16  ep0value;
 
+__xdata u8 usbDeviceSerialNumber[9] = { USB_DEVICE_SERIAL_NUMBER 18 };
```

Comment:
sorry, but i can't have this.
xdata and data bytes are both very very precious.  make whatever you are doing pull directly from the USB SERIAL number.